packages/devtools_app/lib/src/timeline/flutter/timeline_screen.dart

```diff
@@ -0,0 +1,174 @@
+// Copyright 2019 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'package:flutter/material.dart';
+import 'package:flutter_icons/flutter_icons.dart';
+
+import '../../flutter/screen.dart';
+import '../../service_extensions.dart';
+import '../../ui/flutter/label.dart';
+import '../../ui/flutter/service_extension_widgets.dart';
+import '../../ui/flutter/vm_flag_widgets.dart';
+import '../../ui/icons.dart';
+import '../timeline_controller.dart';
+
+class TimelineScreen extends Screen {
+  const TimelineScreen() : super('Timeline');
+
+  @override
+  Widget build(BuildContext context) => const TimelineBody();
+
+  @override
+  Widget buildTab(BuildContext context) {
+    return Tab(
+      text: name,
+      icon: Icon(Octicons.getIconData('pulse')),
+    );
+  }
+}
+
+class TimelineBody extends StatefulWidget {
+  const TimelineBody();
+
+  @override
+  TimelineBodyState createState() => TimelineBodyState();
+}
+
+class TimelineBodyState extends State<TimelineBody> {
+  TimelineMode _timelineMode = TimelineMode.frameBased;
+
+  @override
+  void initState() {
+    super.initState();
+  }
+
+  @override
+  Widget build(BuildContext context) {
+    return Column(
+      children: [
+        Row(
+          crossAxisAlignment: CrossAxisAlignment.center,
```

Comment:
nit: audit named parameters in this method to make sure they aren't at their default values.

Reply:
done.